src/utils/numbers/isZero.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import { BigNumber } from 'bignumber.js'
+
+function isZero(amount: string): boolean {
```

Comment:
Here, we can use `string | number | BigNumber`

Reply:
fixed [here](https://github.com/jibrelnetwork/jwallet-web/pull/441/commits/8ba1ac5589d08555665366aaa738b04ec34e6a2b)